orchestra/gitutils/__init__.py

```diff
@@ -86,8 +86,8 @@ def current_branch_info(repo_path):
     if branch_file.is_file():
         commit = branch_file.read_text().strip()
     else:
-        # Look into .git/info/refs
-        refs = (dot_git / "info" / "refs").read_text().strip()
+        # Look into .git/info/refs and into .git/info/packed-refs
```

Comment:
Typo, it's `.git/info/ref` and `.git/packed-refs`

Reply:
Done!